aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -142,10 +142,10 @@ Tensor per_channel_affine_qtensor_cpu(
     const Tensor& scales,
     const Tensor& zero_points,
     IntArrayRef axis) {
-  Tensor dst = at::_empty_per_channel_affine_quantized_like(
+  Tensor dst = at::_empty_per_channel_affine_quantized(
```

Comment:
what should be the order of arguments?
should we put size in the end so it is closer to options?

Reply:
I think sizes should be first - similarly to how we do it for at::full or at::rand